packages/tpetra/core/src/Tpetra_FECrsGraph_def.hpp

```diff
@@ -281,11 +286,20 @@ void FECrsGraph<LocalOrdinal, GlobalOrdinal, Node>::doOwnedPlusSharedToOwned(con
           flag[indices[j]] = true;
       }
       
-      bool lclSuccess = true;
+      bool success = true;
       for(size_t i=0; i<(size_t)flag.size(); i++)
-        if(!flag[i]) {lclSuccess=false; break;}
+        if(!flag[i]) {success=false; break;}
```

Comment:
Why don't we use a counter instead ofe the boolean `success` and count the number of entries with flag[i]==true? In this way when we error out we can say how many nodes in the column map are not attached to any row.

Reply:
I can do that.